### PR TITLE
feat: document embedded subprocesses

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/bpmn-io/internal-docs/issues/873

## Description

This PR documents collapsed subprocess support in Camunda 8. With Camunda 8.4, Operate shipped support. Only Optimize does not yet support collapsed subprocesses.

image

related to https://github.com/bpmn-io/internal-docs/issues/873

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by nikku — APPROVED** [review]
Reads great.

Love the discrimination against call activities, too.

**Review by christinaausley — COMMENTED** [review]
We say the following:

"...it is not possible to drill down into the subprocesses." "This functionality allows you to simplify the view of a process diagram and facilitates drill-down capabilities to examine details."

I think this is a bit vague. What do you mean by "drill down" into a subprocess? What kinds of things could I do? Should we link to related documentation?

We say "it is not possible to drill down into the subprocesses" but also "This functionality allows you to simplify the view of a process diagram and facilitates drill-down capabilities" and these contradict one another. Can we clarify?

**Comment by nikku** [comment]
Drilldown: https://dictionary.cambridge.org/dictionary/english/drill-down, a well established term in technical tools to "dive one (hierarchy) level deeper" into something.

All tools but optimize fully support this for their actions.
Optimize does not support "drill down", and hence everything that is attached to it (i.e. detailed heatmap on the collapsed sub-process contents.

Does that make sense @christinaausley? How would you rephrase it to be more clear?

**Comment by christinaausley** [comment]
> Drilldown: https://dictionary.cambridge.org/dictionary/english/drill-down, a well established term in technical tools to "dive one (hierarchy) level deeper" into something.
> 
> All tools but optimize fully support this for their actions. Optimize does not support "drill down", and hence everything that is attached to it (i.e. detailed heatmap on the collapsed sub-process contents.
> 
> Does that make sense @christinaausley? How would you rephrase it to be more clear?

This is great feedback! I think saying "...it is **not** possible to drill down into the subprocesses" and then "This functionality allows you to simplify the view of a process diagram and **facilitates drill-down capabilities** to examine details" is what is confusing -- does this permit drill down capabilities, or does it not?

**Comment by nikku** [comment]
> Are we essentially disclaiming in the note that this does not provide drill down capabilities for Optimize, but it does for all other components?

Exactly. The note is simply a heads-up for Optimize.

**Review by christinaausley — APPROVED** [review]

**Review by nikku — APPROVED** [review]
